core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -509,6 +509,7 @@ public void draw(float delta) {
         } else if (isFailed) {
             canvas.beginWithoutCamera(); // DO NOT SCALE
             canvas.draw(border, canvas.getWidth()/2-(border.getRegionWidth()/2), canvas.getHeight()/2-(border.getRegionHeight()/2));
+            displayFont.setColor(Color.CYAN);
```

Comment:
Why?

Reply:
See my PR comment. There was a bug in the fail screen where sometimes the "Game Over" text was Cyan as expected, but sometimes it was white or even like faded gray (idk). Noticed it while running through the game/testing on this branch. It didn't seem worth a whole PR for that one fix, so I just threw it in here. It has nothing to do with the Credits scene, but fixes the other issue. I can make a separate PR for it if you want, but for one line... eh?